tests/profiler/test_profiler.py

```diff
@@ -218,11 +218,11 @@ def test_simple_profiler_summary(tmpdir, extended):
     sep = os.linesep
     max_action_len = len("on_before_batch_transfer")
 
-    for hook in hooks:
+    for i, hook in enumerate(hooks):
         with profiler.profile(hook):
             pass
 
-        profiler.recorded_durations[hook] = [sometime]
+        profiler.recorded_durations[hook] = [sometime + i]
```

Comment:
The output of this test case needs updating if my suggestion gets accepted, but we should speed this up, maybe?
```suggestion
        profiler.recorded_durations[hook] = [sometime + i*0.001]
```